### PR TITLE
add assign group to namespace lab

### PR DESCRIPTION
**Pull request by bartr**
# Purpose of PR

- add assign group to namespace lab
- use :0.9 tag
- remove unused GitOps files

## Type of PR

- [x] Documentation changes
- [x] Labs changes
- [x] Codespaces changes
- [ ] Code changes
- [ ] Test changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,20 +89,35 @@ The first lab is an `inner-loop` lab that introduces Codespaces and the various
...
 
-[Lab](./labs/res-edge-labs/deploy-res-edge/README.md#inner-loop-with-res-edge): Deploy Resilient Edge Data Service to Codespaces
+[Lab](./labs/deploy-res-edge/README.md#inner-loop-with-res-edge): Deploy Resilient Edge Data Service to Codespaces
+
+## Assign a Group to a Namespace
+
+> This lab is a prerequisite for the Kustomize lab
+
+- Res-Edge uses `Groups` to allow applying to multiple clusters at the same time
+- In order for a `Namespace` and associated `Application(s)` to deploy to a Cluster, you must `assign` a Group(s) to the Namespace
```
Comment (mahomedalid) [inline README.md:108]: The format is not consistent, for one side Namespace and Applications(s) are shown betwen backticks, but then later Cluster, Group(s) and Namespaces are not.

File: README.md [hunk 2]
```diff
@@ -89,20 +89,35 @@ The first lab is an `inner-loop` lab that introduces Codespaces and the various
...
 - Connectivity
 
-[Lab](./labs/res-edge-labs/deploy-res-edge/README.md#inner-loop-with-res-edge): Deploy Resilient Edge Data Service to Codespaces
+[Lab](./labs/deploy-res-edge/README.md#inner-loop-with-res-edge): Deploy Resilient Edge Data Service to Codespaces
+
+## Assign a Group to a Namespace
+
+> This lab is a prerequisite for the Kustomize lab
+
+- Res-Edge uses `Groups` to allow applying to multiple clusters at the same time
```
Comment (mahomedalid) [inline README.md:107]: Suggested: "Res-Edge uses `Groups` to allow deploying multiple changes to multiple clusters at the same time" or "applying multiple manifests ...", to try to answer "applying what?"

File: README.md [hunk 3]
```diff
@@ -89,20 +89,35 @@ The first lab is an `inner-loop` lab that introduces Codespaces and the various
...
+- In order for a `Namespace` and associated `Application(s)` to deploy to a Cluster, you must `assign` a Group(s) to the Namespace
+- This lab will demonstrate:
+  - Assigning the `Store Group` to the `imdb Namespace`
```
Comment (mahomedalid) [inline README.md:110]: Store is uppercase, imdb is lowercase, is that on purpose?

**Review by mahomedalid — APPROVED** [review]
Amazing, I can't wait to try this lab with the changes myself. Small comments, nothing crazy.

File: README.md (hunk 3 above)
Comment (bartr) [reply README.md:110]: The official name is IMDb

The actual name in the service is imdb

Should probably use imdb for consistency

**Pull request merged**